awscli/topics/s3-config.rst

```diff
@@ -82,6 +82,12 @@ could instead run these commands::
     $ aws configure set default.s3.use_accelerate_endpoint true
     $ aws configure set default.s3.addressing_style path
 
+To programmatically set these values for a profile other than the default
```

Comment:
In sounds a bit confusing as for me, probably `programmatically`  is redundant here.

Reply:
I used this wording the be consistent with the previous paragraph.